graph/resolver/schema.resolvers.go

```diff
@@ -409,6 +415,92 @@ func (r *mutationResolver) DeleteDiscussionTags(ctx context.Context, discussionI
 	return r.DAOManager.DeleteDiscussionTags(ctx, discussionID, tags)
 }
 
+func (r *mutationResolver) AddDiscussionFlairTemplatesAccess(ctx context.Context, discussionID string, flairTemplateIDs []string) ([]*model.DiscussionFlairTemplateAccess, error) {
+	authedUser := auth.GetAuthedUser(ctx)
+	if authedUser == nil {
+		return nil, fmt.Errorf("Need auth")
+	}
+
+	// Only allow the mod to add discussion tags
+	modCheck, err := r.DAOManager.CheckIfModeratorForDiscussion(ctx, authedUser.UserID, discussionID)
+	if err != nil || !modCheck {
+		return nil, fmt.Errorf("unauthorized")
+	}
+
+	return r.DAOManager.PutDiscussionFlairTemplatesAccess(ctx, discussionID, flairTemplateIDs)
+}
+
+func (r *mutationResolver) DeleteDiscussionFlairTemplatesAccess(ctx context.Context, discussionID string, flairTemplateIDs []string) ([]*model.DiscussionFlairTemplateAccess, error) {
+	authedUser := auth.GetAuthedUser(ctx)
+	if authedUser == nil {
+		return nil, fmt.Errorf("Need auth")
+	}
+
+	// Only allow the mod to delete discussion tags
+	modCheck, err := r.DAOManager.CheckIfModeratorForDiscussion(ctx, authedUser.UserID, discussionID)
+	if err != nil || !modCheck {
+		return nil, fmt.Errorf("unauthorized")
+	}
+
+	return r.DAOManager.DeleteDiscussionFlairTemplatesAccess(ctx, discussionID, flairTemplateIDs)
+}
+
+func (r *mutationResolver) InviteUserToDiscussion(ctx context.Context, discussionID string, inviteeID string) (*model.DiscussionInvite, error) {
+	authedUser := auth.GetAuthedUser(ctx)
```

Comment:
allowing users to invite each other

Reply:
Do we need to make sure that the user who is inviting has access to the conversation _and_ the conversation allows for users to invite other users?

Reply:
tracking here: https://github.com/nedrocks/delphisbe/issues/32